src/main/java/step3/domain/LottoGame.java

```diff
@@ -0,0 +1,40 @@
+package step3.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import step3.view.InputView;
+import step3.view.OutputView;
+
+public class LottoGame {
+    private static final int PRICE_LOTTO = 1000;
```

Comment:
돈과 같은 단위는 언더바를 통해 가독성을 높여볼 수 있을 것 같아요 😄 
```suggestion
    private static final int PRICE_LOTTO = 1_000;
```

Reply:
int 값에 언더바 추가가 가능하단 것을 처음 알았네요..
감사합니다. 수정하겠습니다!